cmd/juju/machine/upgradeseries.go

```diff
@@ -155,34 +155,31 @@ func (c *upgradeSeriesCommand) SetFlags(f *gnuflag.FlagSet) {
 
 // Init implements cmd.Command.
 func (c *upgradeSeriesCommand) Init(args []string) error {
-	numArguments := 3
-
 	if len(args) < 1 {
 		return errors.Errorf("wrong number of arguments")
 	}
 
-	prepCommandStrings := []string{PrepareCommand, CompleteCommand}
-	prepCommand, err := checkPrepCommands(prepCommandStrings, args[0])
+	subCommand, err := checkSubCommands([]string{PrepareCommand, CompleteCommand}, args[1])
```

Comment:
I used "prepCommand" over "subCommand" because name "subCommand" is already taken by an imported package, and it means something very different. But I guess not harm done.

Reply:
Yeah, I changed it because "prepCommand" read (to me) as possibly to do with "prepare", whereas it can be "prepare" or "complete".